tests/benchdnn/graph/utils.hpp

```diff
@@ -85,29 +85,28 @@ enum { CRIT = 0x001, WARN = 0x002, NEED_CLEANUP = 0x004 };
 #define DNN_GRAPH_SAFE(f, s, ss) \
     do { \
         try { \
-            f; \
+            (f); \
         } catch (const dnnl::error &e) { \
-            if ((s & CRIT) || (s & WARN)) { \
+            if (((s)&CRIT) || ((s)&WARN)) { \
```

Comment:
Funny that `&` didn't get separated with spaces...

Reply:
I guess clang-format interprets `&` as an address operator and `(s)` as type cast.